eahub/templates/eahub/edit_profile.html

```diff
@@ -1,81 +1,224 @@
+          </div>
+
+          <div class="mb-5"
+            data-type-name="event_attended"
+            data-type-label="Events attended"
+          >
+            <div class="vue-profile-tag-input"></div>
+          </div>
+
+
+
+          <h2>Career</h2>
+          <div class="row row-cols-2">
+            <div class="col">
+              {% include 'includes/field.html' with form=form field=form.job_title %}
+            </div>
+            <div class="col col-bool-with-margin-top">
+              {% include 'includes/field.html' with form=form field=form.open_to_job_offers %}
+            </div>
+          </div>
+
+          <div class="mb-5"
+            data-type-name="career_interest"
+            data-type-label="Career interests"
+          >
+            <div class="vue-profile-tag-input"></div>
+          </div>
+
+          <div class="mb-5"
+            data-type-name="expertise_area"
+            data-type-label="Expertise areas"
+          >
+            <div class="vue-profile-tag-input"></div>
+          </div>
+
+          {% if profile.expertise_areas_other %}
+            {% include 'includes/field.html' with form=form rows=1 field=form.expertise_areas_other %}
+          {% endif %}
+
+          <div class="mb-5"
+            data-type-name="cause_area"
+            data-type-label="Cause areas"
+          >
+            <div class="vue-profile-tag-input"></div>
+          </div>
+
+          {% if profile.cause_areas_other %}
+            {% include 'includes/field.html' with form=form rows=1 field=form.cause_areas_other %}
+          {% endif %}
+        
+
+          <h2>Settings</h2>
+          {% include 'includes/field.html' with form=form field=form.is_public %}
+          {% include 'includes/field.html' with form=form field=form.allow_messaging %}
+          <a href="{% url 'profiles_app:delete_profile' %}">Delete My Profile</a>
+
+        </div>
+        
+        <div class="col-3 btn-submit__col d-none d-lg-block">
+          <div class="btn-submit__container">
+            <button class="btn btn-lg btn-primary btn-submit" type="submit">Save</button>
+          </div>
+        </div>
+
+      </div>
+
+      <div class="btn-submit__container" data-pos="bottom">
+      <button class="btn btn-lg btn-primary btn-submit" type="submit">Save</button>
+      </div>
+
+    </form>
+
+    {% if settings.DJANGO_ENV.value == "local" %}
+      <script
+        defer
+        src="{{ settings.WEBPACK_DEV_URL }}/component_profile_edit.bundle.js"
+      ></script>
+    {% else %}
+      <script defer src="{% static "dist/component_profile_edit.js" %}"
+      ></script>
+    {% endif %}
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/select2/4.0.12/js/select2.min.js" defer></script>
+    <script src="{% static 'django_select2/django_select2.js' %}" defer></script>
+    {{ form.media.css }}
+  </div>
 
 {% endblock %}
 
-{% block submit%}Update{% endblock%}
+{% block footer %}
```

Comment:
Footer missing

![image](https://user-images.githubusercontent.com/15908208/112799202-f3b8c000-9065-11eb-9611-24db850fefe5.png)

Reply:
for ux reasons as discussed